playbooks/hammer_devel.yml

```diff
@@ -1,8 +1,7 @@
 ---
 - hosts: all
-  become: yes
   roles:
     - etc_hosts
     - git
-    - ruby_scl
+    - { role: ruby_scl, become: yes }
```

Comment:
Can this be:

```
- role: ruby_scl
  become: yes
```

I find the hash syntax a bit jarring in yaml.

Reply:
Yes. Yes it can. :)